frontend/package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "axios": "^1.2.1",
+    "jwt-decode": "^3.1.2",
```

Comment:
you don't need to decode jwt in frontend. if you need data that are not encoded, make your server send them